### ISSUE TITLE
doesn't work on windows

### ISSUE THREAD

**Issue by simoneb** [mafintosh/benny-hill#5]
Fails with

```
$ benny-hill
events.js:160
      throw er; // Unhandled 'error' event
      ^

Error: spawn play ENOENT
    at exports._errnoException (util.js:1007:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:182:32)
    at onErrorNT (internal/child_process.js:348:16)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
    at Function.Module.runMain (module.js:577:11)
    at startup (node.js:160:18)
    at node.js:456:3
```

**Comment by mafintosh** [comment]
do you know a standard command on windows that lets you play an mp3 in the terminal?

**Comment by simoneb** [comment]
see #6

**Comment by mimafogeus2** [comment]
Another sad Windows user here. Is #6 or an alternative going to be merged soon?

**Comment by billiegoose** [comment]
@mafintosh this can be closed thanks to #11 methinks!

**Comment by mafintosh** [comment]
Woot
